config/config.yaml

```diff
@@ -160,11 +150,36 @@ read_lineage_call:
 # variants of concern (protein name -> variants)
 voc:
   S:
-    - N501Y
-    - E484K
-    - K417N
+    - L18F
```

Comment:
I think we should place this before the benchmarking block, not in it. With benchmarking block I mean above the large comment in line 88

Reply:
so also the duplication was on purpose?